src/rdkafka_ssl.c

```diff
@@ -1114,6 +1165,32 @@ int rd_kafka_ssl_ctx_init (rd_kafka_t *rk, char *errstr, size_t errstr_size) {
         }
 #endif
 
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
+        if(rk->rk_conf.ssl.openssl_engine_location && !rk->rk_conf.ssl.ptr_engine) {
+                char* engineLoadErrStr = NULL;
```

Comment:
General code-style is snake_case, use `engine_errstr`

Reply:
sure will change it in next iteration